README.md

```diff
@@ -34,10 +34,10 @@ The project provides two interfaces: the libSQL API, which supports all the feat
 
 To get started with the libSQL API:
 
-* [JavaScript](https://github.com/libsql/libsql-experimental-node)
-* [Rust](core) 
+* [JavaScript](https://github.com/libsql/libsql-js)
+* [Rust](https://github.com/libsql/libsql-client-rs) 
```

Comment:
Hey! The Rust repository you're pointing to is actually something different and you should keep the existing link.